pep-9999.rst

```diff
@@ -253,6 +253,160 @@ specification for `docker-compose`_::
 .. _docker-compose: https://github.com/docker/compose/blob/789bfb0e8b2e61f15f423d371508b698c64b057f/setup.py#L28-L61
 
 
+Compatibility Examples
+======================
+
+The authors of this PEP recognises that various tools need to both read
+from and write to this format for dependency specification. This section
+aims to provide direct comparison with and examples for translating to/from
+the currently used standard, :pep:`508`.
+
+.. note::
+
+        For simplicity and clarity, various ways in which TOML allows you to specify each
+        specification is not represented. These examples use the standard inline representation.
+
+        For example, while following are considered equivalent in TOML, we choose the
+        second form for the examples in this section.
+
+		.. code-block:: toml
+
+			aiohttp.version = "== 3.6.2"
+			aiohttp = { version = "== 3.6.2" }
+
+
+Version Constrained Dependencies
+--------------------------------
+
+**No Version Constraint**
+
+.. code-block::
+
+        aiohttp
+
+
+.. code-block:: toml
+
+        aiohttp = {}
+
+**Simple Version Constraint**
+
+.. code-block::
+
+        aiohttp (>= 3.6.2, < 4.0.0)
```

Comment:
Just as a note, the parentheses are not needed. The normal PEP 508 form is

```
aiohttp >= 3.6.2, < 4.0.0
```

Reply:
Yes, I am aware of that. Was going for the complete form and is often times more readable. Would you recommend that the form without the paranthesis be used in the examples instead? Happy to make that change.

Reply:
The unbracketed form is more common, as far as I'm aware, and I find it more readable. But I find it hard to care, personally, either PEP 508 form looks so overwhelmingly better to me than the TOML form that it's irrelevant.